.gitignore

```diff
@@ -27,3 +27,4 @@ __pycache__
 .DS_Store
 .envrc
 .state/
+venv/
```

Comment:
this should be handled in your global gitignore 
```suggestion
```

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/